rts/Rendering/Fonts/CFontTexture.cpp

```diff
@@ -89,56 +89,23 @@ class FtLibraryHandler {
 		: config(nullptr)
 		, lib(nullptr)
 	{
-		{
-			const FT_Error error = FT_Init_FreeType(&lib);
-
-			FT_Int version[3];
-			FT_Library_Version(lib, &version[0], &version[1], &version[2]);
+		const FT_Error error = FT_Init_FreeType(&lib);
 
-			std::string msg = fmt::sprintf("%s::FreeTypeInit (version %d.%d.%d)", __func__, version[0], version[1], version[2]);
-			std::string err = fmt::sprintf("[%s] FT_Init_FreeType failure \"%s\"", __func__, GetFTError(error));
+		FT_Int version[3];
+		FT_Library_Version(lib, &version[0], &version[1], &version[2]);
 
-			if (error != 0)
-				throw std::runtime_error(err);
-		}
-
-        #ifdef USE_FONTCONFIG
-		if (!UseFontConfig())
-			return;
-
-		{
-			std::string msg = fmt::sprintf("%s::FontConfigInit (version %d.%d.%d)", __func__, FC_MAJOR, FC_MINOR, FC_REVISION);
-			ScopedOnceTimer timer(msg);
-			ZoneScopedNC("FtLibraryHandler::FontConfigInit", tracy::Color::Purple);
-
-			try
-			{
-				FcInit();
-			} catch (const std::exception& e) {
-				LOG("FcInit() runtime error: \"%s\"", e.what());
-				config = nullptr;
-				return;
-			}
-
-			FcConfigEnableHome(FcFalse);
-			config = FcInitLoadConfigAndFonts();
-			if (!config)
-				return;
+		std::string msg = fmt::sprintf("%s::FreeTypeInit (version %d.%d.%d)", __func__, version[0], version[1], version[2]);
+		std::string err = fmt::sprintf("[%s] FT_Init_FreeType failure \"%s\"", __func__, GetFTError(error));
```

Comment:
shouldn't these be under `if (error != 0)`, they don't do anything otherwise?

Reply:
You're right XD. I was only looking at the fontconfig stuff, so didn't notice that part.

Actually, msg isn't even used, looks like it was there for a ScopedTimer, but it was never added. Msg should totally go below that.

Reply:
Done!

Note till now I didn't change the freetype init code so this is a bit out of the PR scope but since I had to retab those lines probably fair game. Not a big deal but just for the record.